README.md

```diff
@@ -2375,6 +2418,65 @@ DESCRIPTION
   Get a MySQLDatabase.
 ```
 
+## `mw database mysql import DATABASE-ID`
+
+Imports a dump of a MySQL database
+
+```
+USAGE
+  $ mw database mysql import DATABASE-ID -i <value> [-q] [-p <value>] [--temporary-user] [--ssh-user <value>] [--gzip]
+
+ARGUMENTS
+  DATABASE-ID  The ID or name of the database
+
+FLAGS
+  -i, --input=<value>           (required) the input file from which to read the dump ("-" for stdin)
```

Comment:
Why is this a flag and not an argument? I assumed that flags should be optional an required input is given as argument.

Reply:
See https://github.com/mittwald/cli/pull/243#discussion_r1556058166